test1.py

```diff
@@ -16,13 +16,23 @@
 del paras['tol']
 print paras
 
+#tuples -- immutable
+lis = (19, 73, -42)
+for n in range(len(lis)):
+    print "lis[%s] = %s" % (n,lis[n])
+
 
 paras = {}
 paras['niter'] = 500
 paras['tol'] = 1.e-06
 for key in paras:
     print "a[%s] = %s" % (key,paras[key])
 
+#sets -- mutable, no order for elements
+myset = set()
```

Comment:
add "physics" also

Reply:
This is already done below. Does that look good?

Reply:
Yes!